Source/Provers/SMTLib/SMTLibProverOptions.cs

```diff
@@ -139,7 +139,7 @@ public override void PostParse()
         case SolverKind.CVC4:
           SolverArguments.Add(
             "--lang=smt --no-strict-parsing --no-condense-function-values --incremental --produce-models");
-          if (Logic == null) Logic = "ALL_SUPPORTED";
+          if (Logic == null) Logic = "ALL";
```

Comment:
@Ying1123 :  This looks great.  Could you please add a comment about the version of CVC5 which is needed for this change. I am also wondering if we should either 
(1) rename SolverKind.CVC4 to SolverKind.CVC5, or
(2) add a different option called SolverKind.CVC5 keeping the original option unchanged.

Do you have any advice?

@bkragl, @RustanLeino , @zvonimir  :  FYI

Reply:
I have added.

Reply:
@Ying1123:   Is there any reason to support both CVC4 and CVC5 in Boogie?  If note, I recommend renaming SolverKind.CVC4 to SolverKind.CVC5 and the command-line option string to "cvc5". If you search for CVC4 and "cvc4" in the Boogie code, you should be able to find all occurrences.

Reply:
I think there is no reason to have both. And I can change it into cvc5.